pycloudlib/ec2/instance.py

```diff
@@ -67,7 +67,7 @@ def image_id(self):
         """Return id of instance."""
         return self._instance.image_id
 
-    def add_network_interface(self):
+    def add_network_interface(self) -> str:
```

Comment:
Let's also type hint _attach_network_interface and document the expected return of a private_ip_address?